### ISSUE TITLE
Newsletter date translation

### ISSUE THREAD

**Issue by arthursp** [magento/magento2#4642]
### Preconditions
Magento 2 is installed
French pack of translation is installed

### Steps to reproduce

- Go to Newsletter templates and start a new queue

### Expected result
When you choose a date with the jquery picker, date is formated on the right DB format

### Actual result
It is formated with the french format and there is an error with wrong date format on server side

**Comment by oserediuk** [comment]
Hello @arthursp ! Looks like this issue belongs to pack which you have installed. I cannot reproduce this case on magento french locale

**Comment by palamar** [comment]
@arthursp could you please provide additional information

**Comment by arthursp** [comment]
Hi,
On back office, when you translate date ( on french in my case) , when its submited, an error appear concerning date format.
Get Outlook for iOS

On Wed, Jun 22, 2016 at 6:05 PM +0200, "Mykola Palamar" notifications@github.com wrote:

@arthursp could you please provide additional information

---

**Comment by veloraven** [comment]
@arthursp could you please specify Magento version used?
Please also provide more detailed steps you did for "Install a french pack of translation" step.
And do you claim that the bug is in the date format which admin sees when he selects date picker, or the way how it saved in the DB?

**Comment by arthursp** [comment]
It was 2.0.4

the pack is imaginaerum : http://imaginaerum.github.io/magento2-language-fr-fr/

I put it on app/i18n/imaginaerum/fr_fr/dictionary.csv

**Comment by arthursp** [comment]
The error is : a:4:{i:0;s:117:"DateTime::__construct(): Failed to parse time string (31 août 2016 13:59:12) at position 0 (3): Unexpected character";i:1;s:5345:"#0 vendor/magento/module-newsletter/Model/Queue.php(185): DateTime->__construct('31 ao\xC3\xBBt 2016 1...')

caused by the fact that the form is sent with french date format.

**Comment by arthursp** [comment]
Maybe it's a timezeone problem , when override /vendor/magento/module-newsletter/Model/Queue.php like this : 

```
/**
     * Set $_data['queue_start'] based on string from backend, which based on locale.
     *
     * @param string|null $startAt start date of the mailing queue
     * @return $this
     */
    public function setQueueStartAtByString($startAt)
    {
        if ($startAt === null || $startAt == '') {
            $this->setQueueStartAt(null);
        } else {
            try{
                $search = array('janv.', 'févr.', 'mars', 'avr.', 'mai','juin', 'juil.', 'août', 'sept', 'oct.', 'nov.', 'déc.');
                $replace = array('Jan', 'Feb', 'Mar', 'Apr', 'May','Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec');
                $startAt = str_replace($search, $replace, $startAt);
                $time = (new \DateTime($startAt))->getTimestamp();                 
            }catch(\Exception $e){
                echo '<pre>';print_r($e->getMessage());exit();
            }            
            $this->setQueueStartAt($this->_date->gmtDate(null, $time));
        }
        return $this;        
    }
```

It's working, maybe you should use DateTime::createFromFormat 
http://php.net/manual/fr/datetime.createfromformat.php

**Comment by PiscesThankIT** [comment]
I meet the same issue. 
Steps to reproduce:
-  Set Account Setting > Interface Locale > Chinese(China)
-  Stores > configuration > General > Locale Options > Local > China
-  Marketing > Newsletter Template  choose a template  do the action "Queue Newsletter"
-  The first field "Queue Date Start" is a  jquery picker in Chinese string, like "2016年10月7日 a2:09:20"
-  When I click the "Save Newsletter", the error happens

```
1 exception(s):
Exception #0 (Exception): DateTime::__construct(): Failed to parse time string (2016年10月7日 a2:09:20) at position 4 (�): Unexpected character
```

If the Interface Locale and Local are all set to be USA, there will be no problem.

**Comment by veloraven** [comment]
Internal Ticket was created MAGETWO-59954

**Comment by miguelbalparda** [comment]
There is a proposed fix for this issue, #10736. Any feedback on this would be much appreciated!

**Comment by magento-engcom-team** [comment]
@arthursp, thank you for your report.
We've created internal ticket(s) MAGETWO-59954 to track progress on the issue.
